messages/checkpoint_event.go

```diff
@@ -0,0 +1,7 @@
+package messages
+
+// CheckpointEvent ...
+type CheckpointEvent struct {
+	Commit  uint64
+	Prepare uint64
```

Comment:
@pvanbuijtene just a small suggestion - can we use `CommitPosition` and `PreparePosition` here to be more explicit?

Reply:
@shaan1337 looks better indeed, it's applied.

Reply:
thanks @pvanbuijtene !